### ISSUE TITLE
Lab Key has empty value

### ISSUE THREAD

**Issue by AmProsius** [AmProsius/gothic-1-community-patch#179]
**Describe the bug**
The Lab Key has the label "Value" in the inventory, but the item has no value.

**Expected behavior**
The Lab Key no longer has a "Value" label.

**Additional context**
The key has a value of 0 and therefor the value isn't displayed. The German description of the key is *Laborschlüssel*.

**Comment by AmProsius** [comment]
https://github.com/AmProsius/gothic-1-community-patch/blob/5533641bc89119b14deed2552352cb783a0e5343/scriptbase/_work/Data/Scripts/Content/Items/MissionItems_3.d#L76-L80

changed to

```d
    TEXT[0]             = "Opens the chest in the alchemy lab";
    TEXT[1]             = "of the swamp camp.";
    //TEXT[1]               = "";
    //TEXT[5]             = NAME_Value;
    //COUNT[5]            = value;
```

**Comment by szapp** [comment]
This bug will be resolved by #176.
